ui/src/OpenSearch.tsx

```diff
@@ -228,4 +229,58 @@ export async function queryOpenSearch(question: string, index_name: string, mode
     console.log("OS query ", JSON.stringify(query))
     var response = sendQuery(query, index_name);
     return response;
+}
+
+export const createFeedbackIndex = async () => {
+    const indexMappings = {
+        "mappings": {
+            "properties": {
+                "interaction_id": {
```

Comment:
do you want to use this as the _id field? Can guarantee single feedback

Reply:
I do that too. I just figure it might be nice to have that explicit when we (eventually) do analytics about this.